.github/workflows/daily-ci.yml

```
@@ -49,12 +51,32 @@ jobs:
           cache-dependency-glob: "uv.lock"
           python-version: ${{ matrix.python-version }}
 
+      - name: Install gcc for macos
+        if: ${{ (matrix.os == 'macos-15') && (matrix.gt4py-module == 'cartesian') }}
+        run: |
+          brew install gcc@14
+          gcc-14 --version
+          echo "CC=gcc-14" >> $GITHUB_ENV
+          echo "CXX=g++-14" >> $GITHUB_ENV
+
+      - name: Install latest-stable xcode for macos
+        if: ${{ (matrix.os == 'macos-15') && (matrix.gt4py-module != 'cartesian') }}
+        uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: latest-stable
+
+      - name: Install libomp for macos
```

Maybe more explicit? 
```suggestion
      - name: Install libomp for macos clang
```